dockerImages.yml

```diff
@@ -28,7 +28,7 @@
 # Tools are sorted alphabetically for ease of use.
 
 biowdl-input-converter: "quay.io/biocontainers/biowdl-input-converter:0.2.1--py_0"
-pyyaml: "biowdl/pyyaml:3.13-py37-slim"
 minimap2: "quay.io/biocontainers/minimap2:2.17--h84994c4_0"
+pyyaml: "biowdl/pyyaml:3.13-py37-slim"
```

Comment:
I think we replaced this with the input converter. So the pyyaml image can be removed.